parsec/_parsec.pyi

```diff
@@ -255,3 +255,25 @@ class ChunkID:
     def from_bytes(bytes: bytes) -> ChunkID: ...
     def from_hex(hex: str) -> ChunkID: ...
     def new() -> ChunkID: ...
+
+from rust_typing.invite import (
```

Comment:
You should instead create `_parsec` folder with `__init__.pyi` & `invite.pyi` inside (with `__init__.pyi` containing this file)

Reply:
done